doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -319,11 +319,11 @@ The updateContext request payload contains a list of contextElement
 elements. Each contextElement is associated to an entity (whose
 identification is provided in the entityId element, in this case we
 provide the identification for Room1) and contains a list of
```

Comment:
"and contains a list of attributes elements."

Looking to the JSON above this lines, maybe a better wording is:

"and contains a vector with a list of attributes."

In addition, note that there is a whitesapce between 's' and '.', that should be removed in the final wording.